xlators/features/compress/src/cdc.c

```diff
@@ -304,13 +304,13 @@ struct volume_options options[] = {
                     "compression ratio; memLevel=9 uses maximum memory "
                     "for optimal speed. The default value is 8."},
     {.key = {"compression-level"},
-     .default_value = "-1",
+     .default_value = "1",
      .type = GF_OPTION_TYPE_INT,
      .description = "Compression levels \n"
                     "0 : no compression, 1 : best speed, \n"
                     "9 : best compression, -1 : default compression "},
     {.key = {"min-size"},
-     .default_value = "0",
+     .default_value = "1024",
```

Comment:
All data is going to be stored in disk blocks, and most probably their size will be 4096 bytes. So any file whose size is up to 4096 bytes will still use one disk block, with or without compression.

But I'm also fine with 1024 if you prefer this value.

Reply:
I was under the impression it's just network compression? I must have missed something - will re-read the code.

Reply:
Oh, you are right. Sorry. I mixed things...

Then there's no issue with the current size.